mono/profiler/coverage.c

```diff
@@ -200,12 +200,11 @@ obtain_coverage_for_method (MonoProfiler *prof, const MonoProfilerCoverageData *
 {
 	g_assert (prof == &coverage_profiler);
 
-	int offset = entry->il_offset - coverage_profiler.previous_offset;
 	CoverageEntry *e = g_new (CoverageEntry, 1);
 
 	coverage_profiler.previous_offset = entry->il_offset;
 
-	e->offset = offset;
+	e->offset = entry->il_offset;
```

Comment:
The comment at the top of the file must be updated for this change.

Reply:
Why do we need to change that? That breaks the API

Reply:
We don't have to, but the old one didn't really made sense to me.